config/core/build-configs.yaml

```diff
@@ -276,6 +276,7 @@ fragments:
       - 'CONFIG_PINCTRL_AMD=y'
       - 'CONFIG_NVME_CORE=y'
       - 'CONFIG_BLK_DEV_NVME=y'
+      - 'CONFIG_SERIAL_8250=y'
```

Comment:
Maybe add it at the top of the list before `CONFIG_SERIAL_8250_DW=y`?

Reply:
Done